docs/interpreting-experiments/confidence-intervals.md

```diff
@@ -1,34 +1,266 @@
+
+#### Pros of sequential analysis
+
+- **You don't have to predetermine your sample size.** Although it is still useful to use Eppo's [Sample Size Calculator](../planning-experiments/index.md) to understand, operationally, when you can expect to be able to detect an effect if it exists, unlike with fixed-sample analysis, sequential analysis does not *require* you to do a power analysis beforehand. More importantly, it also does not require you to restart the experiment if any of the parameters of that power analysis (such as the expected metric values and variances) end up being incorrect.
+- **You can make any decision at any time, safely.** Go ahead, peek away 👀! If a metric is marked as statistically significant (so the confidence interval is entirely above or below zero), you don't have to worry about it being a false positive just because you haven't reached a predetermined sample size. Also, you can decide to keep running the experiment to collect more sample, and, unlike with fixed-sample analysis, doing so will not invalidate the statistical guarantees.
+- **Any data you collect can be useful, even if something goes wrong.** If a data problem means that you can't use data past a certain date, you can at least use all the data you've collected up until then. This means no need to rerun an experiment just because a pipeline broke or there was a bug in logging.
+
+#### Cons of sequential analysis
+
+- **You might have less power.** Compared to a fixed-sample analysis, sequential analysis has less power, meaning it's less likely to detect a real effect. However, this loss in power is mostly concentrated in really small sample sizes, and it can be mitigated by using techniques like [CUPED](cuped.md). Furthermore, fixed-sample analysis only has more power if the power analysis used to set the predetermined sample size is actually correct: if the treatment group has a higher than expected variance for a given metric, for example, you'll end up being less powered than you anticipated—and the only thing you can do in that case is rerun the experiment from scratch. With sequential analysis, if you have less power than you'd like you can always choose to keep running the experiment to collect more sample, without affecting the statistical guarantees
```

Comment:
> Compared to a fixed-sample analysis, sequential analysis has less power

I think it may be useful to be a bit more explicit here: for a fixed amount of time, it has less power

Reply:
> However, this loss in power is mostly concentrated in really small sample sizes, and it can be mitigated by using techniques like [CUPED](cuped.md)

The loss in power is true for all sample sizes no?

Also, CUPED helps but then again CUPED helps fixed sample too so in the end it's still worse no?